guide/src/reference/passing-rust-closures-to-js.md

```diff
@@ -63,6 +63,12 @@ return but the JavaScript closure still needs to be valid!
 For this scenario, you need the `Closure` type, which is defined in the
 `wasm_bindgen` crate, exported in `wasm_bindgen::prelude`, and represents a
 "long lived" closure.
+The `Closure` type is currently behind a feature which needs to be enabled:
```

Comment:
Oh sorry I meant to comment on this earlier, but this isn't quite accurate in the sense that `Closure` itself is not behind a nightly feature, only `Closure::new` is. Stable users need to use `Closure::wrap`